src/pages/Dashboard/Withdrawal.tsx

```
@@ -1,37 +1,19 @@
-];
-
+import { useGetAccountDetailsQuery} from "../../slices/accountApiSlice";
 const Withdrawal = () => {
   const [step, setStep] = useState(1);
-  // const [page, setPage] = useState(1);
   const [withdrawalMethod, setWithdrawalMethod] = useState(""); 
-  const [cardData, setCardData] = useState<any>([]);
-	const [selectedCard, setSelectedCard] = useState<any | undefined>({});
-  const [isTokenComplete, setIsTokenComplete] = useState(false);
-  const [token, setToken] = useState<string[]>(Array(8).fill(""));
+  const [_isTokenComplete, setIsTokenComplete] = useState(false);
```

ESLint

[error] '_isTokenComplete' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)